recipes/webdavclient2/meta.yaml

```diff
@@ -0,0 +1,60 @@
+
+{% set name = "webdavclient2" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "797f121c4e381cb57d7f46abef513191da74959d70096e435269d9479a71ee7f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
+  # noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - requests
+    - lxml
+    - libxml2
```

Comment:
I don't think the package explicitly needs `libxml2` and `libxslt`, just a working `lxml` install.